README.md

```diff
@@ -19,38 +22,44 @@ To install the `SquareMobilePaymentsSDK` into your Xcode project, follow these s
 
 Optionally, you can also add the `MockReaderUI` product to your target to simulate a physical reader when one is not present in a sandbox environment.
 
-> [!WARNING]  
-> Please note that the `MockReaderUI` product should only be used in debug configurations and is not to be included in the release version of your application.
+#### Option B. CocoaPods
 
-### Cocoapods
-
-Install with [CocoaPods](http://cocoapods.org/) by adding the following to your Podfile:
+Install with [CocoaPods](http://cocoapods.org/) by adding the following to your `Podfile`:
 
 ```
 use_frameworks!
 
 pod "SquareMobilePaymentsSDK", "~> 2.0.0.beta1"
 
 # Optionally include MockReaderUI if you wish to simulate a physical reader when one is not present.
-# This feature is only available when provided a sandbox application id.
+# This feature is only available when provided a sandbox application ID.
 pod "MockReaderUI", "~> 2.0.0.beta1", configurations: ['Debug']
 ```
-_Note that MockReaderUI framework **requires** that `SquareMobilePaymentsSDK` framework to also be present in podfile_
+_Note that MockReaderUI framework **requires** the `SquareMobilePaymentsSDK` framework to also be present in your `Podfile`._
 
-#### Add build phase to setup the SquareMobilePaymentsSDK framework ####
+### 2. Add build phase to setup the framework
 
-On your application targets’ Build Phases settings tab, click the + icon and choose New Run Script Phase. Create a Run Script in which you specify your shell (ex: /bin/sh), add the following contents to the script area below the shell:
+On your application targets’ `Build Phases` settings tab, click the + icon and choose `New Run Script Phase`. Create a Run Script in which you specify your shell (ex: /bin/sh), and add the following contents to the script area below the shell:
 ```
 SETUP_SCRIPT=${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}"/SquareMobilePaymentsSDK.framework/setup"
 if [ -f "$SETUP_SCRIPT" ]; then
   "$SETUP_SCRIPT"
 fi
 ```
 
-## Documentation Links
-* [MobilePaymentsSDK Overview](https://developer.squareup.com/docs/mobile-payments-sdk)
+Make sure the above run script is below any `[CP] Embed Pods Frameworks` or `Embed Frameworks` Build Phase.
```

Comment:
nit: "Make sure this build phase is after any `[CP] Embed Pods Frameworks` or `Embed Frameworks` Build Phase."

Reply:
updated